packages/manager/apps/pci-databases-analytics/src/pages/services/[serviceId]/pools/_components/addEditConnectionPool.tsx

```diff
@@ -0,0 +1,342 @@
+                      disabled={
+                        isEdition || isPendingAddPool || isPendingEditPool
+                      }
+                      {...field}
+                    />
+                  </FormControl>
+                  <FormMessage />
+                </FormItem>
+              )}
+            />
+            <FormField
+              control={form.control}
+              name="databaseId"
+              render={({ field }) => (
+                <FormItem className="flex flex-col gap-1 mt-2">
+                  <FormLabel>
+                    {t('formConnectionPoolFieldDatabaseLabel')}
+                  </FormLabel>
+                  <FormControl>
+                    <Select value={field.value} onValueChange={field.onChange}>
+                      <SelectTrigger>
+                        <SelectValue />
+                      </SelectTrigger>
+                      <SelectContent>
+                        {databases.map((option) => (
+                          <SelectItem key={option.id} value={option.id}>
+                            {option.name}
+                          </SelectItem>
+                        ))}
+                      </SelectContent>
+                    </Select>
+                  </FormControl>
+                  <FormMessage />
+                </FormItem>
+              )}
+            />
+            <FormField
+              control={form.control}
+              name="mode"
+              render={({ field }) => (
+                <FormItem className="flex flex-col gap-1 mt-2">
+                  <FormLabel>{t('formConnectionPoolFieldModeLabel')}</FormLabel>
+                  <FormControl>
+                    <Select value={field.value} onValueChange={field.onChange}>
+                      <SelectTrigger>
+                        <SelectValue />
+                      </SelectTrigger>
+                      <SelectContent>
+                        {Object.values(
+                          database.postgresql.connectionpool.ModeEnum,
+                        ).map((option, index) => (
+                          <SelectItem key={index} value={option}>
+                            {option}
+                          </SelectItem>
+                        ))}
+                      </SelectContent>
+                    </Select>
+                  </FormControl>
+                  <FormMessage />
+                </FormItem>
+              )}
+            />
+            <FormField
+              control={form.control}
+              name="size"
+              render={({ field }) => (
+                <FormItem>
+                  <FormLabel>{t('formUserFieldSizeLabel')}</FormLabel>
+                  <FormControl>
+                    <SizeInput
+                      value={field.value}
+                      onChange={(newValue) => form.setValue('size', newValue)}
+                      min={POOL_CONFIG.size.min}
+                      max={POOL_CONFIG.size.max}
```

Comment:
I think you should be able to get the min and max from the field directly
```suggestion
                      {...field}
```

Reply:
Does not work as I want so check will be done on submit :) 
I also removed the custom input component and replaced it by an input.